### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by amun-sihra**
Add React Frontend - Styling is a WIP.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://nanuda.herokuapp.com",
```
Comment (amun-sihra) [inline frontend/package.json:5]: @geooff I was running into CORS issues and had to do this to bypass it locally. Is there a way you can enable CORS on your end?

File: frontend/src/components/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+												</div>
+											))}
+										</div>
+                    </form>
+                </div>
+            )
+    }
+}
+
+export default Home
```
Comment (amun-sihra) [inline frontend/src/components/Home.jsx:62]: Weird formatting, I'll fix this later on

File: frontend/package.json (hunk 1 above)
Comment (geooff) [reply frontend/package.json:5]: I think so, let me look into this a bit. Is CORS stuff blocking this PR?

**Review by geooff — COMMENTED** [review]
@amun-sihra This commit should address the CORS issues. I added an env_var in case FE ends up on another app

**Pull request closed without merging**